homeassistant/components/keba/sensor.py

```diff
@@ -57,6 +64,9 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
             "total_energy",
             "mdi:gauge",
             ENERGY_KILO_WATT_HOUR,
+            DEVICE_CLASS_ENERGY,
```

Comment:
The `icon` should be `None` where a `device_class` is defined

Reply:
Done